### PR TITLE
Feature availability controllers #2

### PR DESCRIPTION
**Pull request by jassalss**
Hi Team ! Availability controllers is almost complete. Please review my PR and share feedback.

### CODE REVIEW

File: server/api/client_secret.json [hunk 1]
```diff
@@ -0,0 +1 @@
+{"web":{"client_id":"240375553858-hfpeurdk1uiek7h8cbrhf13105m8bv5a.apps.googleusercontent.com","project_id":"calend-app","auth_uri":"https://accounts.google.com/o/oauth2/auth","token_uri":"https://oauth2.googleapis.com/token","auth_provider_x509_cert_url":"https://www.googleapis.com/oauth2/v1/certs","client_secret":"l-tnOowsFhPcUhkayJuc3VJW","redirect_uris":["http://127.0.0.1:5000/sign-in-with-google","http://127.0.0.1:5000/signMeIn","http://localhost:5000/authorize","http://localhost:5000/sign-in-with-google","http://127.0.0.1:5000/authorize","http://localhost:3000/","http://localhost:3000/authorize","http://localhost:3000/after-login/"],"javascript_origins":["http://localhost:3000","http://127.0.0.1:3000"]}}
```
Comment (freddy2659) [inline server/api/client_secret.json]: Is this meant to be commited? Should the secret part of it be in the .env?

**Review by freddy2659 — CHANGES_REQUESTED** [review]
Pretty good, just double the secret bit

File: server/api/client_secret.json (hunk 1 above)
Comment (jassalss) [reply server/api/client_secret.json]: sorry. I added this file just for experiment which didn't go well. I forget to remove this file

**Review by freddy2659 — APPROVED** [review]
Awesome

**Pull request merged**